lib/mayaUsd/ufe/Global.cpp

```diff
@@ -292,27 +295,24 @@ MStatus initialize()
     // - If the object is a point instance, use the point instance handler.
     Ufe::Transform3dHandler::Ptr lastHandler
         = MayaUsd::ufe::UsdTransform3dFallbackMayaXformStackHandler::create();
-    lastHandler = MayaUsd::ufe::UsdTransform3dMatrixOpHandler::create(lastHandler);
-    lastHandler = MayaUsd::ufe::UsdTransform3dCommonAPIHandler::create(lastHandler);
+    lastHandler = UsdUfe::UsdTransform3dMatrixOpHandler::create(lastHandler);
+    lastHandler = UsdUfe::UsdTransform3dCommonAPIHandler::create(lastHandler);
     lastHandler = MayaUsd::ufe::UsdTransform3dMayaXformStackHandler::create(lastHandler);
-    lastHandler = MayaUsd::ufe::UsdTransform3dPointInstanceHandler::create(lastHandler);
+    lastHandler = UsdUfe::UsdTransform3dPointInstanceHandler::create(lastHandler);
 #ifdef UFE_V4_FEATURES_AVAILABLE
-    lastHandler = MayaUsd::ufe::UsdTransform3dReadHandler::create(lastHandler);
+    lastHandler = UsdUfe::UsdTransform3dReadHandler::create(lastHandler);
 #endif
-    handlers.transform3dHandler = lastHandler;
+    usdUfeHandlers.transform3dHandler = lastHandler;
```

Comment:
Create the Transform3d handler in MayaUsd which contains two extra Maya specific handlers.